Makefile

```diff
@@ -135,6 +135,10 @@ install-core: all
 	done
 	printf "libexecdir = '%s'\nsysconfdir = '%s'\n" \
 	    "$(libexecdir)" "$(sysconfdir)" > $(DESTDIR)$(pythondir)/_stbt/vars.py
+	for filename in $(INSTALL_PYLIB_FILES); do \
+	    [ -x "$$filename" ] && mode=0755 || mode=0644; \
+	    $(INSTALL) -m $$mode $$filename $(DESTDIR)$(pythondir)/$$filename; \
+	done
 	chmod 0644 $(DESTDIR)$(pythondir)/_stbt/vars.py
```

Comment:
The new code needs to go 1 line lower -- you've split up the code that deals with `_stbt/vars.py`.